landlab/components/flow_director/flow_director_steepest.py

```diff
@@ -155,6 +260,18 @@ def __init__(self, grid, surface="topographic__elevation"):
         self.method = "D4"
         super(FlowDirectorSteepest, self).__init__(grid, surface)
         self._is_Voroni = isinstance(self._grid, VoronoiDelaunayGrid)
+
+        # create a : 'flow__link_direction' field if it does not exist yest
+        if "flow__link_direction" not in self._grid.at_link:
+            self._flow__link_direction = grid.add_field(
+                "flow__link_direction",
+                grid.zeros(at="link", dtype=int),
+                at="link",
+                dtype=int,
+            )
+        else:
+            self._flow__link_direction = grid.at_link["flow__link_direction"]
+
```

Comment:
I think you're using the name "*flow__link_direction*" to be compatible with existing names, is that right? Since it is not a CSDMS Standard Name, I think we should drop the double underscore. In my opinion, it just confuses things. The same goes for "*flow__link_direction_at_node*" and "*flow__link_incoming_at_node*".

In addition, I don't think we should *ever* use double underscores in python variable names (i.e. `self._flow__link_direction`) even if they are proper Standard Names.

Should the data type for *flow__link_direction* be `int`? Could it, instead, be `int8`?

Reply:
Good point about the names. Will change. 

Also, I think it could be int8 for sure!